mdd.py

```diff
@@ -256,16 +256,31 @@ def get_boot_info():
 
 def get_cpu_info():
     logging.info("...get cpu info")
-
-    cpu_model = (
-        [
-            line
-            for line in get_command_output("cat /proc/cpuinfo").split("\n")
-            if "model name" in line
-        ][0]
-        .split(":")[1]
-        .strip()
-    )
+    cpu_model = "unknown"
+    aarch64 = platform.machine() == "aarch64"
+    try:
+        if aarch64:
+            cpu_model = (
+                [
+                    line
+                    for line in get_command_output("cat /proc/cpuinfo").split("\n")
+                    if "Model" in line
```

Comment:
I see only the difference in what value we look for. Either `Model` or `model variant`. The whole `cpu_model` code gets copied per architecture. What if we just for both values and just use what get detected first? Sure we can also check for the architecture in parallel. https://www.geeksforgeeks.org/check-multiple-conditions-in-if-statement-python/

Reply:
Don't know if either of those are better:

```
        cpu_model = (
            [
                line
                for line in get_command_output("cat /proc/cpuinfo").split("\n")
                if aarch64:
                    if "Model" in line
                else
                    if "model name" in line
            ][0]
            .split(":")[1]
            .strip()
        )
```

More compact:

```
        cpu_model = (
            [
                line
                for line in get_command_output("cat /proc/cpuinfo").split("\n")
                any(model in line for name in ('Model', 'model name'))
            ][0]
            .split(":")[1]
            .strip()
        )
```

Reply:
With lscpu call now no repetition anymore.